### PR TITLE
The Next Web clone project

### PR DESCRIPTION
**Pull request by jrai0792**
-Initialize the project with index.html, style.css, and linters setup.
-Style basic structure.
-Add the HTML structure.
-Finish style basic structure.
-Started with media queries.
-Added content and style to the navbar and gallery section.
-Add element and style to the latest news.
-Add content and style for the Most popular section.
-Add content and style to the footer section and hover effect.
-Add content and style to the latest deals and add content to the footer.
-Add changes to media queries.
-Add media queries for the gallery, latest news, and funding section.
-Add media queries for the nav-bar, deals, and footer section.
-Add media queries for a tablet screen.
-Updated the README.md file.

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+.world a {
+  text-decoration: none;
+}
+
+.f1-details1 a,
+.f1-details2 a {
+  text-decoration: none;
+  font-size: 0.8125em;
+}
+
```
Comment (kitpao) [inline assets/style.css:113]: - [ ] You are repeating the same rules in different parts of your CSS. For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
image. You are repeating rules as `text-decoration: none;`, `display: none;`, `list-style: none;`, etc

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @jrai0792 and @Adewunmi97 

The organization in your CSS code is pretty nice 💯 and it is a nice touch you took the care to add hover effects 🌟 Please check below to see required changes 🔽 

**_Note:_** All the **[OPTIONAL]** are **not required** in order to approve your project but are personal suggestions to improve your coding experience

- [ ] **[OPTIONAL]** Some of your images look stretched out, you can correct this problem by adding `object-fit: cover` in their styling, this way the proportions of the image will be respected

- [ ] Please Edit the title of your pull request to be more meaningful and related to the project as [requested in pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/), like in the example:
image

- [ ] For next time, remember to write your commits in the imperative tense, as stated in [the article pointed in pathwright](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

### About your implementation

- [ ] CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
    - No small breakpoint as it is mobile-first and the base styles are for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
    - medium: min-width around 768px
    - large: min-width 1024px
So that you have the minimum trouble fixing this, as a personal suggestion I would recommend you that first, you put all desktop-related code inside a media query with min-width of 1024 and test that everything works fine, then proceed to make the movements to put your mobile-related code inside the default CSS.

- [ ] **OPTIONAL** Note that the sizing in your elements get weird because you are using vh units for the height, so when the height is small,  your elements overlap. I would recommend using em, rem or px for heights. For widths, it's ok to use vw because you are already taking care of them with your media queries

### Mobile screen:
- [ ] Please check and fix the following:
image
image
image
image

### Tablet screen:
- [ ] Please check and fix the following:
image
image
image
image

### Desktop screen:
- [ ] Please check and fix the following:
image

You did nicely on desktop 🏅 

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

File: assets/tablet.css [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    width: 85%;
+    float: right;
+    height: 3.8vh;
+    align-items: center;
+    background-color: #000;
+  }
+
+  .fixed1 {
+    display: flex;
+    width: 100%;
```
Comment (bertil291utn) [inline assets/tablet.css:39]: ## Using the Same Rule Repetitively[1]
- [ ] You already have `display:flex` pro[erty, please use it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags


**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: assets/style.css [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+header {
+  background-color: #fff;
+  height: 6vh;
+}
+
+main {
+  background-color: white;
+  margin-top: 7.6vh;
+  height: 608.8vh;
```
Comment (bertil291utn) [inline assets/style.css]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `background-color: white;` and `background-color: #fff;` properties in entire CSS file, and create just one global for both of them
- [ ] Add your HTML tags


**Example:**

**CSS file**

```CSS
.white-background{
  background-color: #fff;
}
```
**Html file**

```HTML
<div class="white-background">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
## **Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### General requirements 

- [ ] Remove extra padding when you large screen approach   

image

### Specific Req.

| Section/Screen                             | Large(1024px) | Medium(768px) | Small |
| :----------------------------------------- | :---: | :----: | :---: |
| Navigation bar                             |   1    |        |       |
| Top 3 articles section                     |       |        |       |
| Latest news section                        |    2   |        |       |
| Latest Funding section                     |   3    |        |    7   |
| Categories (Apps, Gear, Tech, etc) section |   4    |        |       |
| Latest deals section                       |    5   |        |       |
| Footer                                     |       |        | 8      |
###### This table is given each section with his screen width approach, each issue is given by number below this table

1. About links should be more in the left

image

2. Remove extra padding you have with 1024px screens

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

3. Links  should be after the title 

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

4. Each article is overlapping, please fix that

image

5. Latest deals section is overlapping with footer

image

6. Please fix all medium screens approach, elements should have an order
image

7. Same this section is overlapping to another one

image

8. The copyright is missing

image

### OPTIONAL

- It'd be great if you use just one CSS file adding media queries in the same file

```CSS
/* all my rules */

@media (min-width 1024){
}

```

### Summary

- The complete page (small, medium and large screens) has overlapping section issues, please fix them
- Medium screens approach is completely messed, please fix all medium screens approach 
-  Don't forget to work with inline comments below this comment box.

***

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: index.html [hunk 4]
```diff
@@ -7,11 +7,13 @@
     <title>The next web clone</title>
     <link rel="stylesheet" href="assets/normalise.css">
     <link rel="stylesheet" href="assets/style.css">
+    <!-- <link rel="stylesheet" href="assets/smallscreen.css"> -->
+    <!-- <link rel="stylesheet" href="assets/bigscreen.css"> -->
```
Comment (MuriloRoque) [inline index.html]: Please, remove the commented code if you are not using it.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes Requested :red_circle: 

## Hello @jrai0792 and @Adewunmi97 :wave: 

## Thank you for fixing the requested changes from the previous TSEs, you did an amazing job :clap: :clap: :clap: 

## However, we still have somethings we need to change, please check below:

1) Your about links still need some adjustments. This is how they look like from 1025px to 1100px:

Screenshot from 2020-08-13 14-19-31
<br>

2) Latest deals section is still overlapping with the Footer on Medium and Large screens:

Screenshot from 2020-08-13 14-31-42
<br>
Screenshot from 2020-08-13 14-31-53

<br>

3) You still need to fix the ordering in this section for Medium screens, they are also overlapping:

Screenshot from 2020-08-13 14-35-28

<br>

4) In small screens, this section is still overlapping with another:

Screenshot from 2020-08-13 14-39-06

5) In small screens, parts of your navbar are overlapping with some sections of your page, please fix the alignment:

Screenshot from 2020-08-13 14-42-38


<br>

## Summary
- The entire page (small, medium and large screens) has overlapping section issues, please fix all of them;
- Before submitting for another review, please make the requested changes.
- Please check my comment in your code bellow and good luck!

**Comment by kitpao** [comment]
Friendly tip: most of your images are overlaping because you use vh, never use vh to position things or they will be moving everytime.

Vh is 'view height', it is a % of the screen height, so everytime your page is viewed in a different screen size (in this case TSE's different screens) they will be displayed differently, and you won't be able to predict everyone's screen size. For that reason, use px or rem or em to position elements

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey girls, good job making the previously required changes, your project it's looking really good, and we're almost done, however, in order to accomplish as the actual page, please implement the changes listed below:
 
 - [ ] Your code is still overlapping from between 768px to 1024px screens, kindly fix that 💪
review
 
**Please make the required changes and resubmit for a new code review once you are ready**

As always, I wish you all the very best and see you both at the top🚀

  

<hr>

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by lmaldonadoch — CHANGES_REQUESTED** [review]
# Changes Required 🥇 🏃‍♂️

## Great job, team!!! 🏆

You have put a lot of effort into this project, we can tell! There are still some requirements that we have not met from previous reviews that are required for your project to be approved. Here we go:

- [ ] [REQUIRED] On the navbar, the yellow banner is still not implemented, please implement it.
image

- [ ] [REQUIRED] On medium size, the navbar is still overlapping. Please fix it.
image

- [ ] [REQUIRED] The sizing of the top 3 news is off, make it more like the original page.
image

- [ ] [REQUIRED] The titles of the section below Latest Funding are still overlapping on medium size, please fix this.
image

- [ ] [REQUIRED] The navbar on mobile size is also not correctly implemented. Please fix it.
image

- [ ] [REQUIRED] There is a huge space between the Categories and Latest Deals section. This happens because you have a fixed size for .most-popular on the mobile CSS file. Hint :bulb: avoid using fixed height for sections.
image


- [ ] [OPTIONAL] Your images are a little distorted all around. If you use a div with a background image instead of an image and the appropriate sizing (cover/contain) this can easily be fixed.
image



## You have done an amazing job!



Don't give up! You are so close! It seems to me like you are having a hard time with the navbar and some positioning; don't be afraid to ask for help! We are here to help you, so don't hesitate

Please implement the required changes, and think about the optional ones.

It was a pleasure reviewing your code,

Happy coding! 💻

Slack: Luis Angel Maldoando
[LinkedIn](https://www.linkedin.com/in/lmaldonadoch/)

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
Hello Team 👋 
Status: Changes Required ♻️ 

Well done 👍  You have applied all the necessary changes. Congratulations.
Thanks for your efforts and time.

Just a little arrangement before moving on ⬇️ 

#### **Navbar layout**
yellowbar

**2 things going on here:**

- [ ]  Yellow bar is half of the page's width. It should be full.

- [ ]  The images coming right after the yellow bar and the yellow bar itself using the same area. There should be some amount of margin or padding to push the images down so that the users can see the images as a whole.

That's it.
With all my respect... 🙏

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

### Hello again girls! Good news, your project got approved! Congratulations team! 🐱‍👓
giphy4

_**Great work doing all the requested changes, I'm sure you girls learned a lot while implementing them. Your clone looks nice, you have also followed the best practices throughout the code, so you have my approval to move on!**_ 🥇👌

### You girls can now merge. As always, I wish you the very best and see you both at the top🚀

  

<hr>

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**